torch/testing/_internal/common_methods_invocations.py

```diff
@@ -341,6 +348,60 @@ def __init__(self,
         #   outside a function's domain.
         self._domain_eps = 1e-5
 
+    def sample_sparse_inputs(self, device, dtype, requires_grad=False):
```

Comment:
See sample_inputs_unary above. This should be sample_inputs_unary_sparse and be treated analogously (defined outside the class and the default value for the sparse sample_inputs function)

Reply:
Would it make sense to generalize this function to include `layout` parameter?
The layout could be `torch.strided` or `torch.sparse_coo` (or `torch.sparse_csr` as in https://github.com/pytorch/pytorch/pull/50937).

Reply:
@mruberry - Have made the function analogous
@pearu - Since a generic sample generator will be implemented as another PR; different layouts can be taken care over there